ddtrace/tracer/util.go

```diff
@@ -35,6 +40,9 @@ func toFloat64(value interface{}) (f float64, ok bool) {
 	case uint32:
 		return float64(i), true
 	case uint64:
+		if i > uint64(max) {
```

Comment:
My bad in describing the context. `max` is not inclusive. So please use `>=`.

Reply:
👍